### ISSUE TITLE
Worse result than pretrained model

### ISSUE THREAD

**Issue by Karl233** [doubleQ2018/Deep-learning-for-contact_map_v2#2]
Hi, following https://github.com/doubleQ2018/Deep-learning-for-contact_map_v2/issues/1#issue-454442800
I use the CCMpred as the only 2d input in the training code by simply changing these lines
https://github.com/doubleQ2018/Deep-learning-for-contact_map_v2/blob/934b551ca3642f1e5ba31e7926cbca934b32f5c0/train.py#L41-L45

to
```
pair_feature = pair_feature[:,:,:,0:1]
input_2d = tf.reshape(pair_feature, (1, seqLen, seqLen, 1))
```

The other configs follow the original settings. However, I am not able to replicate the performance in your pretrained model. I get around 77 and the pretrained model gives me 81. I'm wondering if you made any change in your training code or network? Thank you

**Comment by doubleQ2018** [comment]
I trained the model with the learning rate of 0.01 at first, and then with a smaller learning rate. But I cannot remember the detail.
You can explore some training tricks to improve the performance.
